common/match/create.ts

```diff
@@ -91,3 +94,33 @@ export async function createMatch(pupil: Pupil, student: Student, pool: Concrete
 
     logger.info(`Created Match(${match.uuid}) for Student(${student.id}) and Pupil(${pupil.id})`);
 }
+
+export async function createAdHocMeeting(matchId: number, user: User) {
```

Comment:
The only thing I'm not sure of, is where this function is placed. Maybe appointment create could be a better fit. @Jonasdoubleyou any thoughts on this?